.github/workflows/spell.yml

```diff
@@ -25,19 +25,20 @@ on:
             - ".github/.wordlist.txt"
 
 jobs:
-    check-reviewdog:
-        name: Check Spelling - reviewdog
-        runs-on: ubuntu-latest
-        steps:
-            - name: Checkout
-              uses: actions/checkout@v4
-            - uses: reviewdog/action-misspell@v1
-              with:
-                  github_token: ${{ secrets.GITHUB_TOKEN }}
+#    Seems redundant; removed as pyspelling is customized while this is not
+#    check-reviewdog:
```

Comment:
This comment staying in the source code will lose context. Pyspelling is not there anymore, only spellcheck-github-actions.

Reply:
Ok, I'll remove this comment and clarify in some documentation.